app/models/irregular_income_payment.rb

```diff
@@ -0,0 +1,23 @@
+class IrregularIncomePayment < ApplicationRecord
+  ANNUAL_FREQUENCY = 'annual'.freeze
+  STUDENT_LOAN = 'student_loan'.freeze
+  VALID_IRREGULAR_INCOME_FREQUENCY = [ANNUAL_FREQUENCY].freeze
```

Comment:
This is an array (even if it only has one item in it), so let's name it `VALID_IRREGULAR_INCOME_FREQUENCIES` in the plural.